pkg/Microsoft.Private.PackageBaseline/baseline.packages.props

```diff
@@ -323,7 +323,7 @@
       <Version>4.0.0</Version>
     </BaseLinePackage>
     <BaseLinePackage Include="System.Security.Cryptography.X509Certificates">
-      <Version>4.1.0</Version>
+      <Version>4.1.1</Version>
```

Comment:
@bartonjs this isn't exactly what I had in my mind when I said we should do it for cleanliness. @ericstj  is correct that we should avoid bumping this one unless we want to force all our packages to reference the higher version.